app/js/components/visitsAndEncounters.jsx

```diff
@@ -234,16 +234,24 @@ export class VisitsAndEncounters extends React.Component {
   if (start === end) {
     label = start;
   }
-    const columns = [{
-        className:'columnType',
-        id: 'type',
-        Header: 'Type',
-        accessor: rowProps => rowProps.encounter.slice(0, -10),
-        filterMethod: (filter, rows) =>
-        matchSorter(rows, filter.value, { keys: ["type"] }),
-        filterAll: true,
-        
-      },
+    const columns = [
+      {
+        className:"filterDate",
+        id: 'date',
+        Header: 'Date',
+        accessor: rowProps => rowProps.date.split("T")[0],  
+        Filter: ({ filter, onFiltersChange}) =>  
+           <DateRangePicker  startDate={this.state.startDate}
+                             endDate={this.state.endDate} ranges={this.state.ranges} 
+                             onEvent={this.handleEvent} 
+                            >
+              <Button className="selected-date-range-btn" style={{width:'100%'}}>
```

Comment:
declare the `width: '100%'` 'in css since you already have the class `selected-date-range-btn`